src/libraries/System.Private.CoreLib/src/System/Marvin.cs

```diff
@@ -215,6 +215,7 @@ public static int ComputeHash32(ref byte data, uint count, uint p0, uint p1)
             goto DoFinalRoundsAndReturn;
         }
 
+        [Intrinsic]
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
in theory this attribute slows ryujit a bit for no reason, I'd just leave a comment or wrap it with #if mono